### PR TITLE
Fix formatting console-teleprompter.md

### PR DESCRIPTION
**Pull request by badersur**
I've fixed some issues, but left others because:

Some tutorials use inline code for `file names`, but this file does not. In the template file you ask us to "Use _Italics_ for files, folders, paths.." but is not `inline code` or **bold** better for these?

There are: **"Hello World" application**, **Hello World application** and **hello world program**, is it fine?

In **Reading and Echoing the File** section, there is:

> The first feature to add is to read a text file, and display all that text to the console. First, let’s add a text file. Copy the sampleQuotes.txt file from the GitHub repository for this sample into your project directory. **This will serve as the script for your application**.

Is it a script?!

And:

> The compiler generates that **call** when execution reaches the closing brace of the using statement. The compiler-generated **code** ensures that the resource is released even if an exception is thrown from the code in the block defined by the using statement.

Maybe **code** is a **call**?

And, maybe baces from the following block should be removed because we are asked to replace `Console.WriteLine()` **statement**:

``` cs
{ # maybe this brace should be removed!
    Console.Write(line);
    if (!string.IsNullOrWhiteSpace(line))
    {
        var pause = Task.Delay(200);
        // Synchronously waiting on a task is an
        // anti-pattern. This will get fixed in later
        // steps.
        pause.Wait();
    }
} # maybe should be removed
```

When talking about methods, some have parentheses while others don't!

APIs should be linked but I can't do it now.

cc: @BillWagner

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C24%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609242033417149-1073%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C24%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609242055483621-1073%5Cworkflow_report.html).

File: docs/csharp/tutorials/console-teleprompter.md [hunk 1]
```diff
@@ -346,8 +346,8 @@ namespace TeleprompterConsole
 Put that class in a new file, and enclose that class in the
 `TeleprompterConsole` namespace as shown above. You’ll also need to add a `static using`
 statement so that you can reference the `Min` and `Max` method without the
-enclosing class or namespace names. A static using statement imports the
-methods from one class. This is in contrast with the using statements used
+enclosing class or namespace names. A `static using` statement imports the
```
Comment (badersur) [inline docs/csharp/tutorials/console-teleprompter.md]: Or maybe: static `using`!

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C25%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609250732559293-1073%5Cworkflow_report.html).

File: docs/csharp/tutorials/console-teleprompter.md (hunk 1 above)
Comment (mairaw) [inline docs/csharp/tutorials/console-teleprompter.md]: If you're going to style the whole thing as inline code then it should be `using static`. Or probably you could also called this static `using` statement.

Comment (badersur) [inline docs/csharp/tutorials/console-teleprompter.md]: I've wanted to change it to `using static` but I'm waiting your approval :)

**Comment by mairaw** [comment]
Thanks for the changes @badersur.

I think this topic was written when we didn't have a style guide yet. 

> Some tutorials use inline code for file names, but this file does not. In the template file you ask us to "Use Italics for files, folders, paths.." but is not inline code or bold better for these?

We can check. We don't have much of those yet. This was the same style used by the ASP.NET and we're trying to achieve some consistency across teams. @Rick-Anderson, any thoughts on that?

> When talking about methods, some have parentheses while others don't!

On MSDN, our build system would only add the parentheses if we wanted to talk about a specific overload or show the parameters, otherwise the link doesn't show the parentheses. Maybe we should do the same if we're talking about custom methods in our articles too.

 > APIs should be linked but I can't do it now.

It's ok. We have a tracking bug for that so eventually we'll get those linked throughout our content.

I'll let @BillWagner answer the other questions you had.

File: docs/csharp/tutorials/console-teleprompter.md (hunk 1 above)
Comment (mairaw) [inline docs/csharp/tutorials/console-teleprompter.md]: Sounds good then.  😄

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/console-teleprompter?branch=pr-en-us-1073)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C26%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609261903073512-1073%5Cworkflow_report.html).

**Comment by Rick-Anderson** [comment]
File paths should **always** be *itallic*, never ``code``.

Only include () in methods when you are explaining an overload (which is very rare). Method names should **not** include ().

**Comment by mairaw** [comment]
Thanks @Rick-Anderson!

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
> The first feature to add is to read a text file, and display all that text to the console. First, let’s add a text  file. Copy the sampleQuotes.txt file from the GitHub repository for this sample into your project directory. This will serve as the script for your application.

> Is it a script?!

Probably better to say "This will be the text displayed by the application.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Two sets of changes to make:
Removing the curly braces as you suggested earlier.
Removing the parens from method calls in text when not needed to disambiguate overloads.

**Comment by BillWagner** [comment]
@badersur Were you planning to make the last few requested changes?  After that, this would be ready to merge.

**Comment by badersur** [comment]
Unfortunately I'm busy these days and can't do these changes @BillWagner

**Comment by BillWagner** [comment]
@badersur We understand. I will make those last change on this and your other open PRs so we can merge them.

**Comment by BillWagner** [comment]
I've addressed the remaining feedback by cloning this branch, and creating PR #1210.  

Closing this PR.

**Pull request closed without merging**